pkg/server/backend_manager.go

```diff
@@ -85,7 +87,25 @@ type backend struct {
 func (b *backend) Send(p *client.Packet) error {
 	b.mu.Lock()
 	defer b.mu.Unlock()
-	return b.conn.Send(p)
+	errChan := make(chan error, 1)
+	go func() {
+		err := b.conn.Send(p)
```

Comment:
Assuming https://github.com/grpc/grpc-go/issues/1229 is the problem we're seeing, should we be considering a similar fix when calling Send from client -> proxy server?